lib/src/configuration/better_player_data_source.dart

```diff
@@ -40,7 +40,20 @@ class BetterPlayerDataSource {
   ///If empty, then better player will choose name based on track parameters
   final List<String>? hlsTrackNames;
 
-  ///Optional, alternative resolutions for non-hls video. Used to setup
+  ///Should player use dash subtitles
+  final bool? useDashSubtitles;
```

Comment:
I think BetterPlayerDataSource is getting to big right now. I think the best option right now will be to create BetterPlayerDataSourceHlsSettings and BetterPlayerDataSourceDashSettings and encapsulate specifc configuration there.

Reply:
As I say in the pull request  comment:
> Something like "adaptive streaming media sources (amsm)". I mean, for example, in data source, instead of useHlsSubtitles and useDashSubtitles => useAsmsSubtitles.To make it more transparente to the user.I think the best way is to merge the configuration

Reply:
Solved with the unification